app/views/casa_cases/_volunteer_assignment.html.erb

```diff
@@ -9,22 +9,30 @@
             <th>Volunteer Name</th>
             <th>Volunteer Email</th>
             <th>Status</th>
+            <th>Start Date</th>
+            <th>End Date</th>
             <th>Actions</th>
           </tr>
         </thead>
         <tbody>
           <% @casa_case.case_assignments.each do |assignment| %>
             <tr>
-              <td><%= link_to assignment&.volunteer&.display_name, edit_volunteer_path(assignment.volunteer) %></td>
-              <td><%= assignment&.volunteer&.email %></td>
-              <td>
+              <td id="volunteer-name"><%= link_to assignment&.volunteer&.display_name, edit_volunteer_path(assignment.volunteer) %></td>
```

Comment:
ids should be unique on the page. I believe that when there is more than one volunteer shown, this i will not be unique and we will see a chrome console error. Can you check whether this is true, and if so fix it? Maybe something like id=`volunteer-name-${idx}` where idx is brought in by the each loop https://apidock.com/ruby/Enumerator/each_with_index